qa/tasks/mgr/dashboard/test_osd.py

```diff
@@ -51,6 +52,40 @@ def test_scrub(self):
         self._post('/api/osd/0/scrub?deep=True')
         self.assertStatus(200)
 
+    def test_mark_out_and_in(self):
+        self._post('/api/osd/0/mark_out')
+        self.assertStatus(200)
+
+        self._post('/api/osd/0/mark_in')
```

Comment:
Would it make sense to set `mark_in` when cleaning up the test ?

Reply:
That'd look like testing `mark_in` as part of the clean up, wouldn't it? Or do you suggest to separate `test_mark_out_and_in` into two tests, additionally marking the OSD `in` as part of the clean up?

The former feels a little bit odd to me as a clean up tests a feature which isn't tested in a `test_*` method. The latter might mark the OSD in two times (one time as part of the test and one time as part of the clean up), but I could do that.

Reply:
The only reason for doing it would be to force the cluster into a clean state, even if the osd test fail somewhere. 
This is may happen, if you're using the same cluster twice